src/MongoDB.Bson/Serialization/Conventions/StringIdStoredAsObjectIdConvention.cs

```diff
@@ -19,7 +19,10 @@
 namespace MongoDB.Bson.Serialization.Conventions
 {
     /// <summary>
-    /// A convention that sets representation of a string id class member to ObjectId in BSON with a StringObjectIdGenerator.
+    /// A convention that sets the representation of a string id class member to ObjectId in BSON with a StringObjectIdGenerator.
+    /// 
```

Comment:
do we need to have this empty line?

Reply:
No. It was only for readability. But it's removed by the doc comment processor anyway, so I'll remove it also.

Reply:
Done